react/react1/week2/homework/my-app/src/components/TodoList.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { TodoItem } from "./TodoItem";
+import { useState } from "react";
+import { AddTodoItem } from "./AddTodoItem";
+export const TodoList = ({ todos }) => {
+  const [items, setItems] = useState(todos);
+
+  
+  const addTodos = (newTodo) => {

```

Comment:
I suggest you use consistent names. For instance, in this function, you're adding a single todo but you've named it addTodos. A better option would simply be addTodo

Reply:
Yes, thanks